src/runtime/offers-flow.js

```diff
@@ -238,7 +239,10 @@ export class AbbrvOfferFlow {
     // If result is due to requesting offers, redirect to offers flow
     this.activityIframeView_.acceptResult().then(result => {
       if (result.data['viewOffers']) {
-        new OffersFlow(this.deps_, this.options_).start();
+        const isClosable = result.data && result.data['isClosable'] || false;
```

Comment:
Not sure it's a good idea to pass `isClosable` from the iframe. What does the iframe know that we don't already know?

Reply:
Removed this and using the local var instead.